rabbit_feed.gemspec

```diff
@@ -19,13 +19,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   # Gem for interfacing with RabbitMq
-  spec.add_dependency 'bunny', '>= 1.1.9', '< 1.8.0'
+  spec.add_dependency 'bunny', '2.0.0.rc2'
```

Comment:
It's been in rc for a couple of weeks now. There are some fixes that address some longstanding issues with connectivity that have been plaguing us for a while. I'm happy to go with the rc having done quite a bit of testing. Will go back and update the dependency when the final version is eventually released.